.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please remove this file by adding a `.gitignore` file in the root folder and add this code

```
.vscode
```